services/acm/src/it/java/software/amazon/awssdk/services/acm/AwsCertficateManagerIntegrationTest.java

```diff
@@ -40,6 +40,17 @@ public void list_certificates() {
         Assert.assertTrue(result.certificateSummaryList().size() >= 0);
     }
 
+    @Test
+    public void list_certificates_using_oldSigv4aSigner() {
+        AcmClient client = AcmClient.builder()
+                                    .credentialsProvider(StaticCredentialsProvider.create(getCredentials()))
+                                    .overrideConfiguration(c -> c.putAdvancedOption(SdkAdvancedClientOption.SIGNER,
+                                                                                    AwsCrtV4aSigner.create()))
```

Comment:
Missing imports for `SdkAdvancedClientOption` and `AwsCrtV4aSigner` which is causing builds to fail, otherwise lgtm

Reply:
oops, it got missed in my local commit before I pushed. updated now.